src/main/webapp/js/instructorCourseEnrollPage.js

```diff
@@ -10,7 +10,7 @@ var loadUpFunction = function () {
 
     var ENTER_KEYCODE = 13;
     var enrolTextbox; 
-    if ((enrolTextbox	 = $('#enrollstudents')).length){
+    if ((enrolTextbox     = $('#enrollstudents')).length){
```

Comment:
With regards to the existing style issues, we will be fixing them progressively as new rules are added right?